fireflower/models.py

```diff
@@ -102,7 +102,7 @@ class TaskOutput(FireflowerDeclBase):
     __tablename__ = 'task_outputs'
 
     id = Column(Integer, primary_key=True)
-    task_id = Column(String(255), unique=True)
```

Comment:
One thing I learned recently is that there is no performance penalty for using the Text type over String. Can we just make this a Text?